v5/tutorials/topical/wireless-upload.rst

```diff
@@ -0,0 +1,65 @@
+====================================
+Wireless Upload and Hot/Cold Linking
+====================================
+
+.. note:: Wireless Upload requires at least CLI 3.1.4, Kernel 3.1.6, VEXos 1.0.5, and OkapiLib 3.1.7 (if used).
```

Comment:
3.1.7 isn't an OkapiLib version, which version are you referring to?

Reply:
whatever version you want us to- something that supports the hot/cold linking though preferable

Reply:
I am inclined to say v4.0.0. Which may require that I bump odometry to v4.1.0, though that is something I am willing to do to get this stuff out sooner.

Reply:
OK you got time. We can finalize these versions before actually publishing

Reply:
```suggestion
.. note:: Wireless Upload requires at least CLI 3.1.4, Kernel 3.1.6, VEXos 1.0.5, and OkapiLib 3.3.12 (if used).
```